examples/3d/clustered_decal_maps.rs

```diff
@@ -0,0 +1,424 @@
+//! Demonstrates the normal map, metallic-roughness map, and emissive features
+//! of clustered decals.
+
+use std::{f32::consts::PI, time::Duration};
+
+use bevy::{
+    color::palettes::css::{CRIMSON, GOLD},
+    image::ImageLoaderSettings,
+    light::ClusteredDecal,
+    prelude::*,
+    render::view::Hdr,
+};
+use rand::Rng;
+
+use crate::widgets::{RadioButton, RadioButtonText, WidgetClickEvent, WidgetClickSender};
+
+#[path = "../helpers/widgets.rs"]
+mod widgets;
+
+/// The demonstration textures that we use.
+///
+/// We cache these for efficiency.
+#[derive(Resource)]
+struct AppTextures {
+    /// The base color that all our decals have (the Bevy logo).
+    decal_base_color_texture: Handle<Image>,
+
+    /// A normal map that all our decals have.
+    ///
+    /// This provides a nice raised embossed look.
+    decal_normal_map_texture: Handle<Image>,
+
+    /// The metallic-roughness map that all our decals have.
+    ///
+    /// Metallic is in the blue channel and roughness is in the green channel,
+    /// like glTF requires.
+    decal_metallic_roughness_map_texture: Handle<Image>,
+
+    /// The emissive texture that can optionally be enabled.
+    ///
+    /// This causes the white bird to glow.
+    decal_emissive_texture: Handle<Image>,
+}
+
+impl FromWorld for AppTextures {
+    fn from_world(world: &mut World) -> Self {
+        // Load all the decal textures.
+        let asset_server = world.resource::<AssetServer>();
+        AppTextures {
+            decal_base_color_texture: asset_server.load("branding/bevy_bird_dark.png"),
+            decal_normal_map_texture: asset_server.load_with_settings(
+                "textures/BevyLogo-Normal.png",
```

Comment:
```suggestion
                "BevyLogo-Normal.png",
```

I realized I should just provide a suggestion lol.